py-polars/Makefile

```diff
@@ -1,10 +1,28 @@
-
-test:
-	./tasks.sh test
-
+SHELL=/bin/bash
 
 pre-commit:
 	isort .
 	black .
 	mypy
-	flake8 .
\ No newline at end of file
+	flake8 .
+
+pip:
+	@cd wheels
+	pip install --force-reinstall -U polars-*.whl
+
+release-build:
+	maturin build --release -o wheels -i $(which python)
+
+release-install: release-build pip
+
+test:
+	maturin develop
+	pytest tests
+
+test-build:
+	maturin build -o wheels -i $(which python)
+
+test-install: test-build pip
+
+build-and-test: test-install
```

Comment:
and this one. It is redundant with `make test`

Reply:
Done

Reply:
Rolled back in.